src/Model/Error.php

```diff
@@ -13,11 +13,11 @@
 
 class Error extends DataTransferObject
 {
-    public int $status;
+    public ?int $status;
 
-    public string $error;
+    public ?string $error;
 
-    public string $message;
+    public ?string $message;
 
     public ?array $details;
 }
```

Comment:
@snapshotpl Instead of changing the DTO I would rather handle these issues in `\MB\ShipXSDK\Response\HttpResponseProcessor\ErrorProcessor::getPayload`. Could you please change `return new Error($data);` so it always have proper data? (if status is not provided in $data, it can be taken from HTTP response; if $error or $message doesn't exist, let's put empty string)